commands/local_new.go

```diff
@@ -249,6 +248,9 @@ func initCloud(c *console.Context, brand platformsh.CloudBrand, minorPHPVersion,
 		return err
 	}
 
+	if !c.Bool("no-git") {
```

Comment:
@fabpot this early return prevents adding the cloud files to git and breaks the cloud initialization. I think it should be changed to achieve the intended behavior:

```
if c.Bool("no-git") {
    return nil
}
```